pkg/applyconfiguration/testdata/cronjob/api/v1/applyconfiguration/api/v1/cronjobspec.go

```diff
@@ -101,6 +101,7 @@ type CronJobSpecApplyConfiguration struct {
 func CronJobSpec() *CronJobSpecApplyConfiguration {
 	return &CronJobSpecApplyConfiguration{}
 }
+func (b CronJobSpecApplyConfiguration) IsApplyConfiguration() {}
```

Comment:
Urgh, this is wrong, opened https://github.com/kubernetes/kubernetes/pull/132584 for the fix

Reply:
(Nothing to do with your change, just made me aware)